dags/myla311/dag_311_retrieve_update.py

```diff
@@ -18,14 +18,13 @@
 
 filename = "/tmp/myla311.csv"
 
-MY_APP_TOKEN = Variable.get("SOCRATA_APP_TOKEN")
-USERNAME = Variable.get("SOCRATA_USERNAME")
-PASSWORD = Variable.get("SOCRATA_PASSWORD")
-
 
 def retrieve_save_data(**kwargs):
     # Using Authenticated Client:
     # API Document: https://dev.socrata.com/foundry/data.lacity.org/aub4-z9pc
+    MY_APP_TOKEN = Variable.get("SOCRATA_APP_TOKEN")
+    USERNAME = Variable.get("SOCRATA_USERNAME")
+    PASSWORD = Variable.get("SOCRATA_PASSWORD")
```

Comment:
any reason why this needs to be moved?

Reply:
If these variables are not defined in the airflow db, then testing the integrity of these dags fails. I wanted them to be importable without actually running the dag for CI purposes.